Mt 19765 shipping discount (#314)
elasticpath/elasticpath-dev: docs/promotions-builder/promotions-builder-api/promotions-builder-api-overview.mdx

sidebar_position: 1
---

> The Promotions Builder is a rules engine designed to help you create various types of promotions based on customizable conditions and actions. Through flexible rules and strategies, the Promotions Builder allows merchants to build complex promotions tailored to specific business needs.

Reviewer: I'm not sure if we want to call out the Promotions Builder as a rules engine or not (it is kind of that, but also kind of not?).

```suggestion
The Promotions Builder helps users create various types of promotions based on customizable conditions and actions. Through flexible rules and strategies, the Promotions Builder allows merchants to build complex promotions tailored to specific business needs.
```
